services/ui-src/src/components/modals/AddEditProgramModal.tsx

```diff
@@ -31,6 +31,10 @@ export const AddEditProgramModal = ({
     // prepare payload
     const programName = formData["aep-programName"];
     const dueDate = calculateDueDate(formData["aep-endDate"]);
+    const combinedDataArray = formData["aep-combinedData"];
+    const combinedData =
+      combinedDataArray?.[0] ||
+      "No, the report does not contain items for which the state is unable to remove Separate CHIP information";
```

Comment:
Could abstract the "No, the report does not contain items.." out to the global constants file and use there

Reply:
Good idea